includes/embeds/class-amp-facebook-embed-handler.php

```diff
@@ -114,12 +115,7 @@ public function render( $args ) {
 	 */
 	public function sanitize_raw_embeds( Document $dom ) {
 		// If there were any previous embeds in the DOM that were wrapped by `wpautop()`, unwrap them.
-		$embed_nodes = $dom->xpath->query( "//p/{$this->amp_tag}" );
-		if ( $embed_nodes->length ) {
-			foreach ( $embed_nodes as $embed_node ) {
-				$this->unwrap_p_element( $embed_node );
-			}
-		}
+		$this->unwrap_p_element_by_child_tag_name( $dom, Extension::FACEBOOK );
```

Comment:
Might as well reuse this constant in where `$this->amp_tag` is set in the class above, i.e.:

```php
private $amp_tag = Extension::FACEBOOK;
```